lib/rubocop/cop/performance/count.rb

```diff
@@ -27,56 +30,67 @@ class Count < Cop
         COUNTERS = [:count, :length, :size]
 
         def on_send(node)
-          expression, first_method, second_method, third_method = parse(node)
+          expression, selector, params, counter = parse(node)
+          return unless COUNTERS.include?(counter)
 
-          return unless COUNTERS.include?(third_method)
-
-          begin_pos = if SELECTORS.include?(first_method)
-                        return if second_method.is_a?(Symbol)
-                        expression.loc.selector.begin_pos
+          begin_pos = if selector.is_a?(Symbol)
```

Comment:
There's some duplication here. Perhaps this and the `selector_loc` algorithm can be extracted into a common method?